Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs

```diff
@@ -124,7 +125,7 @@ private static void ShowSettingsDialog()
                         case 0:
                             EditorSettings.serializationMode = SerializationMode.ForceText;
                             EditorSettings.externalVersionControl = "Visible Meta Files";
-                            PlayerSettings.virtualRealitySupported = true;
```

Comment:
Is this now superseded by `PlayerSettings.SetVirtualRealitySupported`?

Reply:
yes. i am not calling SetVirtualRealitySupported. i am hoping that a SetStereoRenderMode gets added so i can set all supported platforms at once (dont make the customer reopen to see the dialog)